CHANGELOG.md

```diff
@@ -36,10 +36,10 @@ You can install it as follows:
 `cargo install cargo-contract --forced --locked`
 
 You will also need to use a version of [`pallet-contracts`](https://github.com/paritytech/substrate/tree/master/frame/contracts)
-later than [TODO](https://github.com/paritytech/substrate/commit/todo)
-(January x, 2023) in your node.
```

Comment:
I removed the date here because the https://github.com/paritytech/substrate/tree/polkadot-v0.9.37 branch has since added more commits on top of its previous Jan 11th HEAD